docs/_layouts/post.html

```diff
@@ -0,0 +1,39 @@
+---
+layout: default
+---
+<article class="post h-entry" itemscope itemtype="http://schema.org/BlogPosting">
+
+  <header class="post-header">
+    <h1 class="post-title p-name" itemprop="name headline">{{ page.title | escape }}</h1>
+    <p class="post-meta">
+      <time class="dt-published" datetime="{{ page.date | date_to_xmlschema }}" itemprop="datePublished">
+        {%- assign date_format = site.minima.date_format | default: "%b %-d, %Y" -%}
+        {{ page.date | date: date_format }}
+      </time>
+
+      {% assign authordata = '' | split:'' %}
+        {% for author in site.authors %}
+          {% if author.username == page.author or page.author.first and page.author contains author.username %}
+            {% capture data %}
+             <a href="/author/{{ author.username }}">{{ author.name }}</a>
```

Comment:
```suggestion
             <a href="{{ author.url | relative_url }}">{{ author.name }}</a>
```
Petite suggestion en reprenant le fix de tout à l'heure sur la redirection ;)